report/source/Main/Future_Recommendations.md

```diff
@@ -1,17 +1,66 @@
-
-Push more tooling into their CI/CD pipeline
-In the future, the ideal is for providers to incorporate tools in their CI/CD pipeline which would produce easily verifiable results. We started this with lattix and white source.
-
-Re: Lattix. We established the value of this type of tool, the concept is good. It saves time, helps evaluate complex software without manual source code review, etc.
-
-Lattix
-* look at competitors
-* push the work into the vendor development
-
-
-Architecture review and process assessments can be done incrementally.
-Security claims validation can be combined with verification.
-
-Need to push out:
-Reference architectures
-Sample diagrams
+# Recommendations
+
+We believe the RABET-V pilot was a success, especially given its execution during the difficult 2020 election season.
+
+That said, there is a long way to go to making the RABET-V process a viable, sustainable operation in the U.S. election community. This section lays out recommendations for improving the current process, establishing an operational structure, and enhancing the service offering in the future to provide more value to technology providers and election offices.
+
+The recommendations are organized into categories below.
+
+## Preparation
+
+The pilot served as both a trial run of the RABET-V process and as part of the design process. Because of this approach, we had a limited set of material that Technology Providers could use to prepare their submissions. This was a major inhibitor of efficiency for both the Technology Provider and the Administrator.
+
+To overcome this, the Administrator should create an addendum to the Program Manual or a separate set of documents that provide reference architectures, sample diagrams, and other examples of a quality initial submission. This can evolve over time and should include an annotated version of the application, showing common sticking points, notes and clarifications. It should also have more information about the submission process to encourage Technology Providers to have the appropriate team members ready to engage at the right times, especially for the Process Review.
+
+## Iterations
+
+The challenges of 2020 led to an abbreviated pilot in which we were unable to test re-verifications. While we have high confidence in the time and cost efficiencies of the developed methodologies, as this is a crucial aspect of the RABET-V value proposition, we recommend additional testing to ensure the re-verification process meets expectations for security. That is, a rigorous effort should be made to validate the assumption that testing can be streamlined for smaller changes.

```

Comment:
Not sure what `for security` means.

Reply:
How about this: "as this is a crucial aspect of the RABET-V value proposition, we recommend additional testing of the RABET-V streamlining process to ensure the proposed approach to re-verification meets expectations for security outcomes."